### PR TITLE
Use HTTPS for publishing docs.

### PR DESCRIPTION
**Pull request by jeschkies**

### CODE REVIEW

File: ci.gradle [hunk 1]
```diff
@@ -59,9 +59,10 @@ gitPublishCopy.dependsOn(scaladoc)
 task ci {
     description = "The main target for CI builds."
     group = "CI"
-    if (TravisEnvironment.isMasterBuild()) {
-        dependsOn githubReport, gitPublishPush
-    } else if (TravisEnvironment.isSecureBuild())  {
-        dependsOn githubReport
-    }
+    dependsOn githubReport, gitPublishPush
```
Comment (jeschkies) [inline ci.gradle]: This is just to test `gitPublishPush` in this PR.

File: ci.gradle [hunk 2]
```diff
@@ -59,7 +59,7 @@ gitPublishCopy.dependsOn(scaladoc)
 task ci {
     description = "The main target for CI builds."
     group = "CI"
-    if (TravisEnvironment.isMasterBuild()) {
+     if (TravisEnvironment.isMasterBuild()) {
```
Comment (alenkacz) [inline ci.gradle]: 🤔 why?

Comment (alenkacz) [reply ci.gradle]: btw. we're starting to be pretty tight into Travis here :) just saying...

**Review by alenkacz — APPROVED** [review]
If it's working, LGTM :) 👍

File: ci.gradle (hunk 2 above)
Comment (jeschkies) [reply ci.gradle]: It was commented and I probably forgot the whitespace.

**Review by zen-dog — APPROVED** [review]
🚢

**Pull request merged**